sdk/communication/Azure.Communication.Identity/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 1.0.0-beta.5 (Unreleased)
 ### Breaking
 - CommunicationIdentityClient.IssueToken and CommunicationIdentityClient.IssueTokenAsync are renamed to GetToken and GetTokenAsync, respectively.
+- CommunicationIdentityClient.CreateUserWithToken and CommunicationIdentityClient.CreateUserWithTokenAsync are renamed to CreateUserAndToken and CreateUserAndTokenAsync, respectively. Their return value is also changed from Tuple<CommunictionUserIdentifier, string> to CommunicationUserIdentifierAndTokenResult.
```

Comment:
A small change here, i'm not sure if the spelling matters, but Tuple<CommunictionUserIdentifier> on line 6 can be changed to Tuple<CommunicationUserIdentifier>

Reply:
I updated it